setup.py

```diff
@@ -8,7 +8,7 @@
 requires = [
     'click',
     'cookiecutter',
-    'networkx < 2.4',
+    'networkx >= 2.4',
```

Comment:
`G.nodes` has existed on the previous version, so this constraint is actually not necessary. Let me know if the version constraint needs to be removed.

Reply:
I vaguely remember that the API change occured with 2.0 and 'G.nodes' exists since then. Could you check that and then change the required version accordingly?

Reply:
`G.nodes` has existed since 1.0: https://networkx.github.io/documentation/networkx-1.8/reference/api_1.0.html#nodes-and-nodes-iter.

Reply:
*at least since 1.0, and could be earlier.

Reply:
I removed the version constraint because 1.0 is a very old release, and the majority of anyone using mesa would use a version that is much later.

Reply:
@Corvince any update?

Reply:
Yes, that sounds very good! `G.nodes` changed its return type and `G.node` was depreciated since 2.0, that is properly what I remembered. But I totally agree that we should just leave the version constraint.